client/app/components/sections/ManageAvailability/ManageAvailability.css

```diff
@@ -1,3 +1,26 @@
+@keyframes stroke {
```

Comment:
Initially I though so as well, and even prefixed them. But CSS modules is clever enough to also change those keyframe names, so no need to prefix.